main/main.py

```diff
@@ -1,5 +1,6 @@
 def main():
-    print("eric writes silly variable names")
+    s = "eric writes silly variable names"
+    print(s)
```

Comment:
why are you calling print as a function, isn't it a statement

Reply:
in python 3 it was changed from a statement to a function and screwed up everyone's code